### DISCUSSION TITLE
Scroll the hover box using j or k keys

### DISCUSSION THREAD

**Discussion by black-pwq** [JetBrains/ideavim#853] [Q&A]
Pressing arrow-down and arrow-up can scroll the hover pop-up box after executing the action ShowHoverInfo. It is quite useful to scroll via j and k when viewing javadoc. I tried the following but it doesn't work
```
nnoremap j <Down>
nnoremap k <Up>
```
Let me make the question clearer:
I use this command to simulate mouse hovering `nnoremap gh :action ShowHoverInfo<cr>`. You can try it by typing `gh` on Java classes and will get a pop-window. This pop-up window can be scrolled via the middle button on mouse. It is convenient if I can use keyboard to slide it, e.g., j/k.

Is there any solution?

**Comment by VasoTodorovic** [comment]
Not sure do you mean something like this one,map pop-ups for fixing error
`map <leader>fix <Action>(ShowIntentionActions)`


or do you mean this yellow light bulb suggestion 
![image](https://github.com/JetBrains/ideavim/assets/125747710/4b984c43-6f14-4aeb-bc90-7a13d2b0e2ab)

can you send me pics of that pop-up do you want

**Reply by black-pwq** [reply]
Like this one:
<img width="484" alt="image" src="https://github.com/JetBrains/ideavim/assets/105478488/045811f5-6c19-4bbd-b574-ffee4e332338">
Happens typically when hovering on classes. It would be perfect if I can slide this pop-up just like the editor window (using j/k).

**Reply by VasoTodorovic** [reply]
```
map <leader>show <Action>(QuickJavaDoc)
map <leader>doc <Action>(Activatedocumentation.v2ToolWindow)
```
Also if you want to see what is available to map in IdeaVim type :actoinlist in Intellij, you will get list of all available Action for mapping. You need to do little bit of work to figure out what is what
Mark as answered if this is it

**Reply by black-pwq** [reply]
Thanks for your help, but I want to slide (scroll up/down) the pop-up window using keyboard (e.g. j/k). I updated the issue if the previous is unclear.
